utils/commands/misc.lua

```diff
@@ -359,7 +359,7 @@ local function clear_corpses(cmd)
         return
     end
     local p = player.print
-    if not trusted[player.name] then
+    if not trusted[player.name] and not this.always_allow_clear_corpses then
```

Comment:
It stays trusted only.

Reply:
resolved in 26f34ea8d5d108a6cbd20848b24bc02525c5ff98